### DISCUSSION TITLE
Cannot delete IP pool

### DISCUSSION THREAD

**Discussion by impirecom** [postalserver/postal#2593] [Help with using Postal]
Hello,

I created two IP pools. Assigned a group for first pool, assigned 2 IPs to second pool.

I then removed IP from second pool, tried to delete but got error "postal IP pool cannot be removed because it still has associated addresses".

The mail server has another pool selected. But Postal kept displaying it is using the second pool. Test message show the IP going out correctly via the first pool. But strangely, the server keep saying it is ending using the second pool.

For some reason,  the second pool refused to be deleted. I even removed IP from server and tried setting IP pools = false. Nothing seems to work.

I thought it would be easy to just delete an empty pool. Looks like a bug with Postal

**Comment by impirecom** [comment]
IP rotation also stopped working for the first pool. All messages keep coming out from same IP. Please help.

**Comment by willpower232** [comment]
Are you able to have a look around the database, see if you can see any associations between the database records that either Postal missed deleting or are invisible from the UI?

**Reply by impirecom** [reply]
Thanks Will. How do I check?

**Reply by willpower232** [reply]
I don't have the database to hand right now but you should be able to rummage through the tables in a program like HeidiSQL or Sequel Ace to find references to them
